src/components/v2/appDetails/k8Resource/nodeDetail/nodeDetail.util.ts

```diff
@@ -386,4 +387,22 @@ export const selectStyles = {
         ...base,
         padding: '0 8px',
     }),
-}
\ No newline at end of file
+}
+
+/**
+ * @description This function is used to trim the manifest data by removing the managed fields from the manifest data
+ */
+export const getTrimmedManifestData = (
+    manifestData: ManifestData,
+    returnAsString: boolean = false,
+): ManifestData | string => {
```

Comment:
Suggestion: We can use dynamic typing here

Reply:
Good suggestion but i think it might be overkill for this case